roles/container_runtime/defaults/main.yml

```diff
@@ -23,7 +23,8 @@ openshift_docker_ent_reg: 'registry.access.redhat.com'
 
 openshift_docker_options: False  # str
 openshift_docker_log_driver: False  # str
-openshift_docker_log_options: []
+openshift_docker_log_options:
```

Comment:
We need to set the appropriate docker_log_driver for what the actual default is on RHEL.

Next, we can make this:
```yaml
openshift_docker_log_options_defaults:
  json:
    - "maz-size=50m"

openshift_docker_log_options: "{{ openshift_docker_log_options_defaults[openshift_docker_log_driver] | default([]) }}"

Reply:
Is there a way to pick up the default log-driver or is that something we have to set as well?

Reply:
Probably can set default for openshift_docker_log_driver.

Reply:
@michaelgugino and the default would be journald?